models/iterable__user_campaign.sql

```diff
@@ -33,7 +35,8 @@ with events as (
         {% endfor %}
 
     from events
-    {{ dbt_utils.group_by(n=9) }}
+    where unique_user_key is not null
```

Comment:
Can you elaborate on the need for this where clause?

Reply:
My reasoning is, since this is a user-grain model, I wanted to exclude where there's no user associated. Since this comes from event-based model `iterable__events` rather than a user-based model, perhaps we come across events without a user. though theoretically, each event has a user associated, so `unique_user_key` should never be null. But I felt this was safe to do just in case.

Reply:
Digging more into the docs from these event tracking platforms, I'm pretty sure all events have an associated user. I believe even if the user is unknown when an event is collected, there will be an anonymous id placed. I suppose we can remove this, it was more of a just-in-case since I found it interesting that it's a user-based model but pulls from an event source. I'll remove this